atoma-proxy-service/src/components/grafana.rs

```diff
@@ -0,0 +1,207 @@
+use reqwest::Client;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use thiserror::Error;
+use tracing::instrument;
+
+/// A client for interacting with Grafana
+///
+/// This struct provides methods for querying data from Grafana dashboards.
+#[derive(Clone)]
+pub struct Grafana {
+    /// The URL of the Grafana instance
+    url: String,
+    /// The API token to use for authentication
+    api_token: String,
+    /// The tag to use to filter dashboards
+    dashboard_tag: String,
+}
+
+#[derive(Deserialize)]
+struct DashboardList {
+    uid: String,
+}
+
+#[derive(Deserialize)]
```

Comment:
Any reason you chose to use `String` here over `chrono::DateTime` ?

```rust
#[derive(Deserialize)]
struct Time {
    from: DateTime<Utc>,
    to: DateTime<Utc>,
}
```

Reply:
Because the time is not actually time, it's `grafana time` which looks like `now` `now-1w` and similar. Also it's just passed around, it's not worth the compute time to do anything with it.